packages/extension-polkagate/src/partials/Review.tsx

```
@@ -4,18 +4,19 @@
 import type { SubmittableExtrinsic } from '@polkadot/api/types';
 import type { AccountJson } from '@polkadot/extension-base/background/types';
 import type { ISubmittableResult } from '@polkadot/types/types';
-import type { Proxy, ProxyTypes, TxInfo } from '../util/types';
+import type { PoolInfo, Proxy, ProxyTypes, TxInfo } from '../util/types';
```

Biome (1.9.4)

[error] Do not shadow the global "Proxy" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)